c2corg_ui/static/js/geomdownload.js

```diff
@@ -56,11 +66,21 @@ app.TrackDownloadController = function(ngeoDownload, mapFeatureCollection) {
  * @param {string} mimetype
  * @private
  */
-app.TrackDownloadController.prototype.downloadFeatures_ = function(
+app.GeomDownloadController.prototype.downloadFeatures_ = function(
   format, extension, mimetype) {
   var geojson = new ol.format.GeoJSON();
   var features = geojson.readFeatures(this.featureCollection_);
-  features = features.filter(app.utils.isLineFeature);
+  switch (this.featureType) {
+    case 'Line':
+      features = features.filter(app.utils.isLineFeature);
+      break;
+    case 'Point':
+      // export only the waypoint itself, not its associated features
+      features = [features[0]];
```

Comment:
Good question. I don't know.

For a waypoint, it makes to get only the waypoint itself, not the associated ones. So only the first feature.

For routes or outings, we don't want the associated waypoints either, I think (?). But are we sure the route/track in the first (line) feature is enough? We could have a complex geometry with several line features, couldn't we?